services/MerchStoreService.ts

```diff
@@ -28,7 +28,7 @@ import { OrderModel } from '../models/OrderModel';
 import { UserModel } from '../models/UserModel';
 import Repositories, { TransactionsManager } from '../repositories';
 import { MerchandiseCollectionModel } from '../models/MerchandiseCollectionModel';
-import EmailService from './EmailService';
+import EmailService, { OrderInfo, OrderPickupEventInfo } from './EmailService';
```

Comment:
nit: Why import the types from here? Should the types not be defined in the types file?

Reply:
We had those types there since they were specifically used for the `EmailService` methods, but now that they're being accessed by multiple service classes, we should probably move them to a types file.

Reply:
Actually, I think since those types are only meant to be associated with `EmailService`, it makes more sense to leave it, since otherwise any class can import those types from `../types` and think that it's okay to use, when instead importing from './EmailService` makes it clearer that its meant to be used with `EmailService`.